app/templates/src/static/css/brand-palette.less

```diff
@@ -3,28 +3,67 @@
    Color variables
    ========================================================================== */
 
-// Official Palette as of October 29, 2013 (Brand 2.0)
+// Official Palette current as of April 9, 2015.
+// See http://git.io/vedVh to view the colors in the Design Manual.
```

Comment:
Sure, it's just long, and git.io is GH's shortener so it's not going anywhere, but yeah can be nice to see the URL at a glance.